content/python/concepts/threading/terms/run/run.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: '.run()'
+Description: 'Returns the activity that takes place within a thread.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Async Await'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.run()` method returns the activity that takes place within a thread. It normally executes in the background after the [`.start()`](https://www.codecademy.com/resources/docs/python/threading/start) method is invoked.
```

Comment:
This intro says what it returns and when it is executed, but not what it does. That should be in the summary, as it's the reason for the function to exist.

Reply:
Good point! I actually dug a little deeper and realized that `.run()` doesn't necessarily return anything. Rather, it _represents_ what happens inside the `target` function passed to the constructor of the Thread object. While this is the default behavior, you can customize the `.run()` function to specifically return something.